templates/about.html

```diff
@@ -26,4 +25,4 @@
 Github</a> page.</p>
 </div>
 <script type="text/javascript" src="../js/yucisite.js"></script>
-</body>
\ No newline at end of file
+</body>
```

Comment:
Don't forget to add an extra line-break at the end of the file. Otherwise `git` will think that the last line has changed every time you add/remove lines. If you're using sublime, you can do this automatically on-save.

Open the sublime text settings and add this line: `"ensure_newline_at_eof_on_save": true,`